deps/build.jl

```diff
@@ -151,7 +151,8 @@ const gcc_support = (
     (v"6.5", v"4.8.2"),
     (v"7.0", v"4.9.2"),
     (v"7.5", v"4.9.2"),
```

Comment:
for 7.5 the check is `#if __GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ > 9)`, so you can drop the `.2` patch version